src/sidebar/components/test/group-list-test.js

```diff
@@ -332,4 +332,58 @@ describe('groupList', function () {
     assert.calledWith($window.open, 'https://test.hypothes.is/groups/new',
       '_blank');
   });
+
+  it('should set showGroupMenu=false if third party service and only one group', function () {
+    // Configure third party service.
+    fakeSettings.authDomain = 'example.com';
+    fakeSettings.services = [{
+      authority: 'publisher.org',
+    }];
+    
+    // Configure only one group.
+    const group = [{
+      id: 'h-devs',
+      links: {
+        html: PRIVATE_GROUP_LINK,
+      },
+      name: 'Hypothesis Developers',
+      organization: groupFixtures.defaultOrganization(),
+      type: 'private',
+    }];
+    fakeGroups.all = () => { return group; };
+
+    const element = createGroupList();
```

Comment:
This approach to creating the group list is fine I think since there are only a small number of tests and the extra overhead from unnecessarily rendering the template won't be too large. You asked in Slack though about testing without rendering the template. The idiomatic way to do that in Angular 1.x for components is to use `$componentController`. Have a search through the codebase for examples.

Reply:
As per slack discussion, I'm going to leave this as is for now (test via rendering) so I can add other assertions later to verify the ng-if's result in the rendering of expected components.

Reply:
I have added the rendering tests. I wanted to do a test of the <ul class=dropdown-menu> but in both the w/ and w/o drop down menu cases it had a length of 0 for the list of elements which was a little confusing so I used the <li class=dropdown-menu__row> instead which are inside that <ul>. I'm not sure if lengthof is the best assertion check for this. I was originally trying to use an isempty check but that didn't work so I changed to checking the length.